tests/test_georaster.py

```diff
@@ -557,24 +542,30 @@ def test_copy(self, example: str) -> None:
             assert r.__getattribute__(attr) == r2.__getattribute__(attr)
 
         # Check data array
-        assert geoutils.misc.array_equal(r.data, r2.data, equal_nan=True)
+        assert np.array_equal(r.data, r2.data, equal_nan=True)
```

Comment:
Why not use the __eq__ functionality here? It would be simpler and easier to understand.